apps/desktop/app/app.ts

```
@@ -729,6 +729,23 @@ async function createMainWindow() {
+      }
+      return false;
+    },
+  );
+
+  browserWindow.webContents.session.setDevicePermissionHandler((details) => {
+    if (details.deviceType === 'usb') {
+      return true;
+    }
+    return false;
+  });
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)